addons/TBMod_rhs/CfgMagazines.hpp

```diff
@@ -361,4 +361,48 @@ class CfgMagazines
     {
         mass = 66.12; // 120
     };
+
+    class rhsusf_50Rnd_762x51;
+    class rhsusf_100Rnd_762x51 : rhsusf_50Rnd_762x51 // 100rnd M240 Box M80
+    {
+        descriptionshort = "Caliber: 7.62x51mm NATO<br />Rounds: 200<br />Used in: M240"; // Caliber: 7.62x51mm NATO<br />Rounds: 100<br />Used in: M240
+        displayname = "200rnd 7.62x51mm"; // "100rnd M240 Box M80"
+        displayNameShort = "200rnd M240"; // "M80 Ball"
+        count = 200; // 100
+        mass = 64.68; // 32.34
+    };
+
+    class rhsusf_50Rnd_762x51_m62_tracer;
+    class rhsusf_100Rnd_762x51_m62_tracer : rhsusf_50Rnd_762x51_m62_tracer // 100rnd M240 Box M62 Tracer
+    {
+        descriptionshort = "Caliber: 7.62x51mm NATO<br />Rounds: 200<br />Used in: M240"; // Caliber: 7.62x51mm NATO<br />Rounds: 100<br />Used in: M240
+        displayname = "200rnd 7.62x51mm (Tracer)"; // "100rnd M240 Box M62 (Tracer)"
+        displayNameShort = "200rnd Tracer"; // "M62 Tracer"
```

Comment:
Ok, vergess was ich im Kommentar hier drüber gesagt habe... **DAS ist kriminell** 50 im Classname und 200 in echt 😄

Reply:
Jup, bin bereits selbst davon überzeugt, die Mags auf originaler Größe zu lassen!